crates/ibc/src/core/ics04_channel/context.rs

```diff
@@ -157,8 +157,7 @@ pub trait ChannelReader {
     ) -> Result<Height, ChannelError>;
 
     /// Returns a counter on the number of channel ids have been created thus far.
-    /// The value of this counter should increase only via method
-    /// `ChannelKeeper::increase_channel_counter`.
+    /// The value of this counter increases only via method `ChannelKeeper::increase_channel_counter`.
```

Comment:
Similarly, the previous comment was more accurate

Reply:
[e2ea605](https://github.com/cosmos/ibc-rs/pull/360/commits/e2ea6054fee2be83cd9421fce8b0b21bd92e754b)